styles.css

```diff
@@ -0,0 +1,943 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+
+.index-body {
+  background: no-repeat;
+  background-image: url(./imgs/background-header.jpg);
+  background-size: cover;
+  background-color: #f7f7f7;
+  width: 100%;
+  height: 900px;
+  background-position: top;
+  font-family: "Inter", sans-serif;
```

Comment:
- [ ] It looks like you're using `Inter` font family on your project, I kindly invite you to change to `Cocogoose, Lato`, this is the font required for this project. 🥥

Reply:
Oh yes, now it's changed for Lato font 👍